integration-tests/chopsticks/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "e2e",
+  "version": "0.0.1",
+  "description": "chopsticks integration tests",
+  "main": "src/index.ts",
+  "type": "module",
+  "repository": "git@github.com:KILTprotocol/kilt-node.git",
+  "author": "[\"KILT <info@kilt.io>\"]",
+  "license": "MIT",
+  "dependencies": {
+    "@acala-network/chopsticks": "0.9.12",
```

Comment:
Aren't these all `devDependencies`?

Reply:
True. Updated in: [48626b5](https://github.com/KILTprotocol/kilt-node/pull/614/commits/48626b53ebcdb7e1a8c66170da8c678e479e55b3)